src/lambda/wizards/samDeployWizard.ts

```diff
@@ -100,6 +151,89 @@ class DefaultSamDeployWizardContext implements SamDeployWizardContext {
         return val ? val.uri : undefined
     }
 
+    public async configureParameterOverrides(
+        {
+            templateUri,
+            missingParameters = new Set<string>()
+        }: {
+            templateUri: vscode.Uri
+            missingParameters?: Set<string>
+        }
+    ): Promise<boolean> {
+        if (missingParameters.size < 1) {
+            const prompt = localize(
+                'AWS.samcli.deploy.parameters.optionalPrompt.message',
+                'The template {0} contains parameters. ' +
+                'Would you like to override the default values for these parameters?',
+                templateUri.fsPath
+            )
+            const responseYes = localize(
+                'AWS.samcli.deploy.parameters.optionalPrompt.responseYes',
+                'Yes'
+            )
+            const responseNo = localize(
+                'AWS.samcli.deploy.parameters.optionalPrompt.responseNo',
+                'No'
+            )
+
+            const quickPick = picker.createQuickPick<vscode.QuickPickItem>({
+                options: {
+                    title: prompt
+                },
+                items: [
+                    { label: responseYes },
+                    { label: responseNo }
+                ]
+            })
+            const response = getSingleResponse(await picker.promptUser({ picker: quickPick }))
+            if (response === responseYes) {
```

Comment:
should this be === responseNo to return true? I understand returning true means "carry on with the deployment" but yes means "take me to the editor".

Reply:
✔️